app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,482 @@ def navigation
+                title: "Pop Music in 1980's Berlin",
+                url: "urlhere",
+                is_exhibition: true,
+                bg_image: "sample/thumb-music.jpg"
               },
               {
-                :title =>"Music piracy",
-                :url => "urlhere",
-                :is_exhibition => true,
-                :bg_image =>"sample/thumb-music.jpg"
+                title: "Music piracy",
+                url: "urlhere",
+                is_exhibition: true,
+                bg_image: "sample/thumb-music.jpg"
               }
             ]
           },
-          :news => {
-            :items => news_items,
-            :blogurl => "http://blog.europeana.eu/tag/#{@channel.id}"
+          news: {
+            items: news_items,
+            blogurl: "http://blog.europeana.eu/tag/#{@channel.id}"
           }
         }
       end
-      
+
 
       def navigation_common
         {
-          :global  => {
-            :options => {
-              :search_active => false,
-              :settings_active => true
+          global: {
+            options: {
+              search_active: false,
+              settings_active: true
             },
-            :logo =>{
-              :url => root_url,
-              :text => "Europeana Search"
+            logo: {
+              url: root_url,
+              text: "Europeana Search"
             },
-            :primary_nav =>{
-              :items => [
+            primary_nav: {
+              items: [
                 {
-                  :url => root_url,
-                  :text => "Home",
-                  :is_current =>false
+                  url: root_url,
+                  text: "Home",
+                  is_current: false
                 },
                 {
-                  :url => root_url + "channels/art",
-                  :text => "Channels",
-                  :is_current =>true,
-                  :submenu => {
-                    :items => [
+                  url: root_url + "channels/art",
+                  text: "Channels",
+                  is_current: true,
+                  submenu: {
+                    items: [
                       {
-                        :url  => root_url + "channels/art",
-                        :text => "Art History"
+                        url: root_url + "channels/art",
+                        text: "Art History"
                       },
                       {
-                        :url  => root_url + "channels/music ",
-                        :text => "Music"
+                        url: root_url + "channels/music ",
+                        text: "Music"
                       }
                     ]
                   }
                 },
                 {
-                  :url => "http://exhibitions.europeana.eu/",
-                  :text => "Exhibitions"
+                  url: "http://exhibitions.europeana.eu/",
+                  text: "Exhibitions"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.